src/components/controls/preview.vue

```diff
@@ -504,6 +531,26 @@
             }
         ]
     };
+    const kmValidationErrorProps = ref({
```

Comment:
"ref" is passed to the component. Need to discuss it further, whether to pass ref or save it to a constant variable.